front-end/generate-task-list.js

```diff
@@ -0,0 +1,65 @@
+const addTaskForm = document.querySelector('.task-form');
```

Comment:
This file is definitely not formatted with Prettier (it seems like it's still not configured/installed properly)